website/common/locales/en/groups.json

```diff
@@ -132,7 +132,14 @@
   "possessiveParty": "<%= name %>'s Party",
   "clearAll": "Delete All Messages",
   "confirmDeleteAllMessages": "Are you sure you want to delete all messages in your inbox? Other users will still see messages you have sent to them.",
-  "optOutPopover": "Don't like private messages? Click to completely opt out",
+  "PMEnable": "Enable",
+  "PMEnableFull": "Enable private messages",
+  "PMDisable": "Disable",
+  "PMDisableFull": "Disable private messages",
```

Comment:
FYI just as a note, this feature in the iOS app uses the same phrase "Disable Private Messages", so this website change is consistent with that, which is good!